### PR TITLE
showcase: Add Crispin Porter Bogusky

### PR DESCRIPTION
**Pull request by mikeriley131**

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -3769,3 +3769,16 @@
...
+  main_url: https://cpbgroup.com/
+  description: >
+    We solve the world’s toughest communications problems with the most quantifiably potent creative assets.
+  categories:
+    - Creative
+    - Advertising
+    - Experience Design
+    - Marketing
+  built_by: Crispin Porter Bogusky
+  built_by_url: http://cpbgroup.com/
```
Comment (jgierer12) [inline docs/sites.yml]:
```suggestion
  built_by_url: https://cpbgroup.com/
```

**Comment by jgierer12** [comment]
Awesome site! Thanks for using Gatsby :raised_hands:

File: docs/sites.yml [hunk 2]
```diff
@@ -3769,3 +3769,16 @@
   built_by_url: "https://www.nearform.com/"
   description: >
     Tools to help diagnose and pinpoint Node.js performance issues.
+- title: Crispin Porter Bogusky
+  url: https://cpbgroup.com/
+  main_url: https://cpbgroup.com/
+  description: >
+    We solve the world’s toughest communications problems with the most quantifiably potent creative assets.
+  categories:
+    - Creative
```
Comment (LekoArts) [inline docs/sites.yml]: If you're an Agency, I'd replace it with that.

```suggestion
    - Agency
```

File: docs/sites.yml [hunk 3]
```diff
@@ -3769,3 +3769,16 @@
...
+    - Advertising
+    - Experience Design
```
Comment (LekoArts) [inline docs/sites.yml]:
```suggestion
    - Design
```

**Comment by KyleAMathews** [comment]
There's conflicts now in the file.

**Pull request merged**